docs/notary-signer-config.md

```diff
@@ -0,0 +1,249 @@
+		</td>
+	</tr>
+	<tr>
+		<td valign="top"><code>grpc_addr</code></td>
+		<td valign="top">yes</td>
+		<td valign="top">The TCP address (IP and port) to listen for GRPC
+			traffic.  Examples:
+			<ul>
+			<li><code>":7899"</code> means listen on port 7899 on all IPs (and
+				hence all interfaces, such as those listed when you run
+				<code>ifconfig</code>)</li>
+			<li><code>"127.0.0.1:7899"</code> means listen on port 7899 on
+				localhost only.  That means that the server will not be
+				acessible except locally (via SSH tunnel, or just on a local
+				terminal)</li>
+			</ul>
+		</td>
+	</tr>
+	<tr>
+		<td valign="top"><code>tls_key_file</code></td>
+		<td valign="top">yes</td>
+		<td valign="top">The path to the private key to use for
+			HTTPS. The path is relative to the directory of the
+			configuration file.</td>
+	</tr>
+	<tr>
+		<td valign="top"><code>tls_cert_file</code></td>
+		<td valign="top">yes</td>
+		<td valign="top">The path to the certificate to use for
+			HTTPS. The path is relative to the directory of the
+			configuration file.</td>
+	</tr>
+	<tr>
+		<td valign="top"><code>client_ca_file</code></td>
+		<td valign="top">no</td>
+		<td valign="top">The root certificate to trust for
+			mutual authentication. If provided, any clients connecting to
+			Notary Signer will have to have a client certificate signed by
+			this root. If not provided, mutual authentication will not be
+			required. The path is relative to the directory of the
+			configuration file.</td>
+	</tr>
+</table>
+
+## `storage` section (required)
+
+This is used to store encrypted priate keys.  We only support MySQL or an
+in-memory store, currently.
+
+Example:
+
+```json
+"storage": {
+	"backend": "mysql",
+	"db_url": "user:pass@tcp(notarymysql:3306)/databasename?parseTime=true",
+	"default_alias": "password1"
+}
+```
+
+<table>
+	<tr>
+		<th>Parameter</th>
+		<th>Required</th>
+		<th>Description</th>
+	</tr>
+	<tr>
+		<td valign="top"><code>backend</code></td>
+		<td valign="top">yes</td>
+		<td valign="top">Must be <code>"mysql"</code> or <code>"memory"</code>.
+			If <code>"memory"</code> is selected, the <code>db_url</code>
+			is ignored.</td>
+	</tr>
+	<tr>
+		<td valign="top"><code>db_url</code></td>
+		<td valign="top">yes if not <code>memory</code></td>
+		<td valign="top">The <a href="https://github.com/go-sql-driver/mysql">
+			the Data Source Name used to access the DB.</a>
+			(note: please include "parseTime=true" as part of the the DSN)</td>
+	</tr>
+	<tr>
+		<td valign="top"><code>default_alias</code></td>
+		<td valign="top">yes if not <code>memory</code></td>
+		<td valign="top">This parameter specifies the alias of the current
+			password used to encrypt the private keys in the DB.  All new
+			private keys will be encrypted using this password, which
+			must also be provided as the environment variable
+			<code>NOTARY_SIGNER_&lt;default_alias_value&gt;</code>.</td>
+	</tr>
+</table>
+
+**Required environment variable:**
```

Comment:
I think the default alias needs a better explanation (in order to support key rotation, keys get created with a field that says what password they have been encrypted with, bla bla)